pkg/controllers/tagging/metrics.go

```diff
@@ -0,0 +1,56 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package tagging
+
+import (
+	"k8s.io/component-base/metrics"
+	"k8s.io/component-base/metrics/legacyregistry"
+	"sync"
+)
+
+var register sync.Once
+
+var (
+	workItemLatency = metrics.NewHistogramVec(
+		&metrics.HistogramOpts{
+			Name:           "cloudprovider_tagging_controller_work_item_latency",
+			Help:           "workitem latency of workitem being in the queue and time it takes to process",
```

Comment:
Based on the [best practices](https://prometheus.io/docs/practices/naming/), the label name should include the unit:
`cloudprovider_tagging_controller_work_item_latency_seconds` or `cloudprovider_tagging_controller_work_item_duration_seconds` might be better since its the total time of the process?

Reply:
will update to `cloudprovider_tagging_controller_work_item_duration_seconds`